lib/x86_cpu/x86_types.ml

```diff
@@ -111,7 +111,7 @@ type opcode =
   | Shiftd of binop * typ * operand * operand * operand
   | Rotate of binop * typ * operand * operand * bool (** left or right, type, src/dest op, shift op, use carry flag *)
   | Bt of typ * operand * operand
-  | Bs of typ * operand * operand * direction
+  | Bs of typ * operand * operand * direction * bool
```

Comment:
This module is exposed to the public interface so changing it will require us to bump BAP's version to 3.0.0.

Could we do this without breaking the interface? I see a few options, like handling them on the modern lifter, but I would first like to hear your opinion.

Reply:
I think the prefix information is passed to the `to_ir` function, so I could infer it that way.